src/razor/src/codeLens/razorCodeLensProvider.ts

```diff
@@ -144,7 +145,7 @@ export class RazorCodeLensProvider extends RazorLanguageFeatureBase implements v
             // We now have a list of references to show in the CodeLens.
             const count = remappedReferences.length;
             codeLens.command = {
-                title: count === 1 ? '1 reference' : `${count} references`,
+                title: count === 1 ? l10n.t('1 reference') : l10n.t('{count} references', count),
```

Comment:
Is this supposed to be:
```suggestion
                title: count === 1 ? l10n.t('1 reference') : l10n.t('{0} references', count),
```
?